templates/observability-remote-write-proxy-route.yml

```diff
@@ -0,0 +1,22 @@
+# This file contains an OpenShift template that creates a route helping to
```

Comment:
This route is only used for development purposes.
For our stage and production environment different routes will be created and configured